python/lsst/rubintv/production/rubinTv.py

```diff
@@ -530,10 +530,13 @@ def dataIdToMetadataDict(butler, dataId, keysToRemove):
         rawmd = butler.get('raw.metadata', dataId)
         obsInfo = ObservationInfo(rawmd)
         d['airmass'] = obsInfo.boresight_airmass
+        d['focus_z'] = obsInfo.focus_z.value
 
-        if 'FOCUSZ' in rawmd:
-            d['focus_z'] = rawmd['FOCUSZ']
-        if 'SEEING' in rawmd:
+        d['Altitude'] = 'null'  # altaz_begin is None when not on sky so need check it's not None first
```

Comment:
Assigning a string to something that should be a float does make me wince a bit when thinking about a `mypy` future. Is there a reason this is not `None` or `NaN`?

Reply:
Yeah, fair enough. I think `None` is the right answer here, I just need to check that that does indeed get rendered as `null` like everything else on the front end. I'll do that tomorrow morning once we're not on sky, and will merge with that change assuming it works. I don't think any of this will ever be mypyed though.

Reply:
`None` works perfectly. Merging now.